style/ruby/README.md

```diff
@@ -42,6 +42,8 @@ Ruby
 * Prefer `protected` over `private` for non-public `attr_reader`s, `attr_writer`s,
   and `attr_accessor`s.
 * Order class methods above instance methods.
+* Do not include a space after the opening brace of a hash or before the closing
+  brace.
```

Comment:
Would this apply to procs/blocks/lambdas as well? In the example code you apply this rule to a block even this guideline explicitly only mentions hashes.

Thoughts on:

* Changing the wording to something along the lines of "all uses of curly braces"
* Explicitly mentioning that block/procs/lambdas are also affected
* Only applying this rule to hashes and not changing the block example